src/frontend/apps/standalone_site/src/features/Header/Header.tsx

```diff
@@ -64,7 +77,7 @@ const Header = forwardRef<Nullable<HTMLDivElement>>((_props, ref) => {
         <LogoIcon width={117} height={80} />
       </Box>
       <Box direction="row" align="center" gap="small" justify="end">
-        <Text>John Doe</Text>
+        <Text>{fullName}</Text>
```

Comment:
I think I would have not use useEffect and directly that:
```suggestion
        <Text>{currentUser?.full_name || ''}</Text>
```

Reply:
It is not possible to do that because of the `currentUser` type. I have this error: `roperty 'full_name' does not exist on type 'User | AnonymousUser'.`